app/routes.py

```diff
@@ -0,0 +1,17 @@
+from app import app
+from flask import render_template
+from app.models import Item
+
+@app.route("/")
+@app.route("/home")
+def home_page():
+    dataframe = Item.query.all()
+    return render_template('home.html', items=dataframe)
+
+@app.route("/FAQ")
+def FAQ_page():
+    return render_template('FAQ.html')
+
+@app.route("/Profile")
+def Profile_page():
+    return render_template('Profile.html')
```

Comment:
Looks like `Profile.html` doesn't exist in the templates, which causes a TemplateNotFound error when opened. Has a template been created? If not, a blank page can be created for now, just so there are no errors in the initial setup.

Reply:
added empty profile page which can be accessed with either:
/profile
/profile/someusername